packages/cli/src/ucc.spec.ts

```diff
@@ -332,25 +307,20 @@ gcp:
   ] = await uccGetAndValidate(filename, "gcp");
 
   expect(userClusterConfig).toEqual({
-    "cert_issuer": "letsencrypt-staging",
-    "controller_image": CONTROLLER_IMAGE_DEFAULT,
-    "data_api_authentication_disabled": false,
-    "data_api_authorized_ip_ranges": [
-      "0.0.0.0/0",
-    ],
-    "env_label": "unit-tests",
-    "log_retention_days": 14,
-    "metric_retention_days": 30,
-    "node_count": 3,
-    "tenants": [
-      "prod",
-      "dev",
-    ],
+    cert_issuer: "letsencrypt-staging",
+    controller_image: CONTROLLER_IMAGE_DEFAULT,
+    data_api_authentication_disabled: false,
+    data_api_authorized_ip_ranges: ["0.0.0.0/0"],
+    env_label: "unit-tests",
+    log_retention_days: 14,
+    metric_retention_days: 30,
+    node_count: 3,
+    tenants: ["prod", "dev"]
   });
   expect(infraConfigAWS).toBeUndefined();
   expect(infraConfigGCP).toEqual({
-    "machine_type": "test-type",
-    "region": "test-region",
-    "zone_suffix": "test-suffix",
+    machine_type: "test-type",
+    region: "test-region",
+    zone_suffix: "test-suffix"
```

Comment:
When I see all these changes I understand that we need to more carefully communicate how we set up our VSCode dev environment.